science/proxmark3-iceman/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                1.0
+PortGroup                 github 1.0
+PortGroup                 qt5 1.0
+
+name                      proxmark3-iceman
+
+github.setup              RfidResearchGroup proxmark3 4.13441 v
+checksums                 sha256 9fd8d6e017d941c403a8f3d0d2f3ab1f9623b82de58dd7aa841d0471b310bd52 \
+                          rmd160 923a587f44bbfad3ee92f3aad91fcaa16c5c9f4b \
+                          size 24568220
+
```

Comment:
you added the `github.tarball_from archive`, but did not update the `checksums` so now it will fail there (I just checked this locally). You probably have the archive downloaded already, so to notice this you'll need to clean the port first and then try again, i.e.: `sudo port clean --all proxmark3-iceman && sudo port -dv checksum proxmark3-iceman`